Functions/Helper.cs

```diff
@@ -8,8 +8,83 @@ namespace Basic
 {
     internal class Helper
     {
-        /*
-         * TODO: Add any needed helper functions
-         */
+        public static void ManageAssignment1()
```

Comment:
Maybe it wasn't clear, but `Helper` file is only here to assist in implementing logic for the assignments. It is not meant to be called first, only the functions in `Solution`.